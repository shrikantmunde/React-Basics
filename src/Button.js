import React from 'react';

const Button = (props) => {
  return (
    <div>
      <p onClick={props.dontClick}>Click me</p>
      <button onClick={props.dontClick}>Click me for something to happen</button>
    </div>
  );
};

export default Button;
