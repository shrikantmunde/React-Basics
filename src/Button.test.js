import React from 'react';
import Button from './Button';
import renderer from 'react-test-renderer';

it('renders a button', function() {
  const component = renderer.create(<Button onClick="My Title" />);
  expect(component).toMatchSnapshot();
})