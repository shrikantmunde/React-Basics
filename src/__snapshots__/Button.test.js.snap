exports[`test renders a button 1`] = `
<div>
  <p
    onClick={undefined}>
    Click me
  </p>
  <button
    onClick={undefined}>
    Click me for something to happen
  </button>
</div>
`;
